src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -238,10 +238,7 @@ public static string ValidateIssuer(string issuer, SecurityToken securityToken,
                 foreach (string str in validationParameters.ValidIssuers)
                 {
                     if (string.IsNullOrEmpty(str))
-                    {
-                        LogHelper.LogInformation(LogMessages.IDX10235);
```

Comment:
Should we still have something logged in this case (this string just seems like the wrong string).

This does seem like something that indicates that something odd is happening, if there are empty strings in the ValidIssuers collection something has gone awry correct?

Reply:
IMHO, we just need the valid issuer is logged here.

Reply:
@ciaozhang i agree with Tim having a null or empty string is unusual.